notebook/db_util.py

```diff
@@ -1,5 +1,7 @@
 # Db util class which take care of executing queries.
 
+from deming.edge import device_types
+from deming.models import EnterpriseUser, KernelSession, EdgeDevice
```

Comment:
curious. this will throw an err to import this before django is setup, right?

Reply:
Formatting the document bought them up ....reverting it